desc/optimize/_constraint_wrappers.py

```diff
@@ -607,9 +618,8 @@ def _set_eq_state_vector(self):
         self._args = self._eq.optimizable_params.copy()
         for arg in ["R_lmn", "Z_lmn", "L_lmn", "Ra_n", "Za_n"]:
```

Comment:
maybe commenting here why we do this (these are parts of the state which our ForceBalance constraint projection (i.e. `eq.solve`) are responsible for handling, and they are a consequence of that projection, thus we don't want them considered in our DOFs (they are effectively removed by the ForceBalance constraint)